### PR TITLE
feat (frontend): reformulate terms and conditions message

### PR DESCRIPTION
**Pull request by charline-rageade**
#4656 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by charline-rageade** [comment]
I have two questions : 
*  do I have to put points at the end of the sentences
*  I don't know if i have compare my branch with develop or another branch for this PR ? (ticket is in 3.8.2 board ?)

File: frontend/i18next.scanner/de/translation.json [hunk 1]
```diff
@@ -472,5 +471,6 @@
   "New email:": "Neue E-Mail:",
   "Type your password:": "Geben Sie Ihr Passwort ein:",
   "Alex Doe": "Alex Müller",
-  "alex.doe@jco.co": "alex.muller@alm.de"
+  "alex.doe@jco.co": "alex.muller@alm.de",
+  "In order to finish your sign in, please read the following usage conditions and accept them": "Um Ihre Registrierung abzuschließen, lesen Sie bitte die folgenden Nutzungsbedingungen und akzeptieren Sie diese"
```
Comment (MariePrandstetter) [inline frontend/i18next.scanner/de/translation.json:475]: Um Ihre Anmeldung abzuschließen, lesen Sie bitte die folgenden Nutzungsbedingungen und akzeptieren Sie diese

Comment (charline-rageade) [reply frontend/i18next.scanner/de/translation.json]: @MariePrandstetter fixed !

**Review by gbonaspetti — APPROVED** [review]

**Comment by gbonaspetti** [comment]
> * do I have to put points at the end of the sentences

I don't think is needed

> * I don't know if i have compare my branch with develop or another branch for this PR ? (ticket is in 3.8.2 board ?)

develop is fine, 3.8.2 is the current version in development

**Comment by grignards** [comment]
Tested on branch.

**Pull request merged**